Automattic/jetpack
# Instant Search: update input box and icon styles

Related to issues reported in https://github.com/Automattic/jetpack/issues/14608; complementary PR to https://github.com/Automattic/jetpack/pull/14661

#### Changes proposed in this Pull Request:
* Update styling of main search box.
  * Ads `:hover` and `:focus` styles.
  * Using `px` units instead of `rem` to better control the output.
  * Clearer separation of input and icon.
  * Uses Gridicon/search icon.
* Revert search icon back to 24px.
* Fix bug with Gridicon's viewBox values reported in https://github.com/Automattic/jetpack/pull/14629#issuecomment-585474931.
* Remove magnifying glass options from Customizer as per https://github.com/Automattic/jetpack/issues/12540#issuecomment-586319510

![jps-search-box-v2](https://user-images.githubusercontent.com/390760/74660847-d4771000-518e-11ea-9daa-ea8a43104521.gif)


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:
1. Follow [these setup instructions](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions) to set up Jetpack Instant Search on your site.
2. Search your site to trigger the Jetpack Search overlay.
3. Ensure that the Jetpack Search input field shows the icon and that both work correctly on all devices.

#### Proposed changelog entry for your changes:
* None.
